invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -69,9 +69,9 @@ def _process_lambda_file(self, fileobj):
         date_format = "YYYY-MM-DD HH:mm:ss.SSSSSS"
         date_format_no_millis = "YYYY-MM-DD HH:mm:ss"
         time_shift = current_app.config["VOCABULARIES_ORCID_SYNC_SINCE"]
-        if self.since:
-            time_shift = self.since
         last_sync = arrow.now() - timedelta(**time_shift)
+        if self.since:
+            last_sync = arrow.get(self.since)
```

Comment:
@sakshamarora1 I'm working on removing the string casts in https://github.com/inveniosoftware/invenio-jobs/issues/87, maybe we won't need the `arrow.get` call as it's already a Python datetime? Not sure how arrow works in general so I could be wrong.